Doc/library/glob.rst

```diff
@@ -23,8 +23,11 @@ according to the rules used by the Unix shell, although results are returned in
 arbitrary order.  No tilde expansion is done, but ``*``, ``?``, and character
 ranges expressed with ``[]`` will be correctly matched.  This is done by using
 the :func:`os.scandir` and :func:`fnmatch.fnmatch` functions in concert, and
-not by actually invoking a subshell.  Note that unlike :func:`fnmatch.fnmatch`,
-:mod:`glob` treats filenames beginning with a dot (``.``) as special cases.
+not by actually invoking a subshell.
+
+Note that files beginning with a dot (``.``) can only be matched by :mod:`glob`
+patterns that also start with a dot,
```

Comment:
This could optionally be a bit more explicit, like

'... patterns that also start with a dot (so ``*.`` will not match), unlike ...'

Reply:
I think this is clear enough as is.